src/main/java/kappzzang/jeongsan/domain/BaseEntity.java

```diff
@@ -0,0 +1,29 @@
+package kappzzang.jeongsan.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.MappedSuperclass;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.time.LocalDateTime;
+
+@Getter
+@SuperBuilder
+@MappedSuperclass
+@NoArgsConstructor
```

Comment:
P2: 무분별한 객체 생성을 방지하기 위해 엔티티 생성자가 public보단 protected로 하는 것이 더 좋다고 생각합니다.

```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```

Reply:
네, 베이스 엔티티는 서브 클래스를 통해서 객체가 생성되도록 해야겠네요. 반영하도록 하겠습니다